src/test/kotlin/apply/ui/api/ApplicantRestControllerTest.kt

```diff
@@ -132,8 +136,8 @@ internal class ApplicantRestControllerTest(
     @Test
     fun `기존 지원자 정보와 일치하지 않는 지원자 생성 및 검증 요청에 대하여 unauthorized 응답을 받는다`() {
```

Comment:
```suggestion
    fun `기존 지원자 정보와 일치하지 않는 지원자 생성 및 검증 요청에 응답으로 Unauthorized를 반환한다`() {
```

아래와 위의 테스트 메서드명과 어조를 맞춰보았습니다.